### PR TITLE
Added Photoview

### PR DESCRIPTION
**Pull request by Skayo**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [X] I have tested the template using the method described in README.md thoroughly
- [X] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [X] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [X] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [X] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/photoview.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        environment:
+            PHOTOVIEW_DATABASE_DRIVER: mysql
+            PHOTOVIEW_MYSQL_URL: $$cap_mariadb_user:$$cap_mariadb_pass@tcp(srv-captain--$$cap_appname-mariadb)/$$cap_mariadb_db
+            PHOTOVIEW_LISTEN_IP: 0.0.0.0
+            PHOTOVIEW_LISTEN_PORT: 80
+            PHOTOVIEW_MEDIA_CACHE: /app/cache
+            MAPBOX_TOKEN: $$cap_mapbox_token
+        volumes:
+            - $$cap_appname-media-cache:/app/cache
+            - $$cap_photos_path:/photos:ro
```
Comment (githubsaturn) [inline public/v4/apps/photoview.yml:29]: `:ro` suffix isn't parsed by CapRover. Must be removed.

**Review by githubsaturn — COMMENTED** [review]
Looks good! Just one minor comment.

File: public/v4/apps/photoview.yml (hunk 1 above)
Comment (Skayo) [reply public/v4/apps/photoview.yml]: Done.

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**